buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf

```diff
@@ -61,6 +61,15 @@
   "server": {
     "sync_skill_settings": false
   },
+  "stt": {
+    "module": "mycroft",
+    "ovos-stt-plugin-vosk-streaming": {
+        "model": "http://alphacephei.com/vosk/models/vosk-model-small-en-us-0.15.zip"
```

Comment:
this would be downloaded if needed, but the image already includes the data in the corresponding xdg directory